### PR TITLE
ConceptworldCorporation.RecentX version 5.0.22.0

### PR DESCRIPTION
**Pull request by heymaad**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/34108)

### CODE REVIEW

File: manifests/c/ConceptworldCorporation/RecentX/5.0.22.0/ConceptworldCorporation.RecentX.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: ConceptworldCorporation.RecentX
+PackageVersion: 5.0.22.0
```
Comment (OfficialEsco) [inline manifests/c/ConceptworldCorporation/RecentX/5.0.22.0/ConceptworldCorporation.RecentX.installer.yaml:5]:
```suggestion
PackageVersion: 5.0
```

File: manifests/c/ConceptworldCorporation/RecentX/5.0.22.0/ConceptworldCorporation.RecentX.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: ConceptworldCorporation.RecentX
+PackageVersion: 5.0.22.0
```
Comment (OfficialEsco) [inline manifests/c/ConceptworldCorporation/RecentX/5.0.22.0/ConceptworldCorporation.RecentX.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 5.0
```

File: manifests/c/ConceptworldCorporation/RecentX/5.0.22.0/ConceptworldCorporation.RecentX.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.4.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: ConceptworldCorporation.RecentX
+PackageVersion: 5.0.22.0
```
Comment (OfficialEsco) [inline manifests/c/ConceptworldCorporation/RecentX/5.0.22.0/ConceptworldCorporation.RecentX.yaml:5]:
```suggestion
PackageVersion: 5.0
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
The PackageVersion needs to match the `A`dd or `R`emove `P`rograms entry.
Remember to update the version folder name.
This can also be done by pressing `.` inside this PR which opens a online version of VS Code

**Review by ghost — CHANGES_REQUESTED** [review]
Please sign the CLA: https://cla.opensource.microsoft.com/microsoft/winget-pkgs?pullRequest=34108

**Comment by Mav3r1x** [comment]
@ItzLevvie2 CLA says gives two options to agree; either I'm the owner or submitting on behalf of an employer. But neither is the case. I just wanted to submit this app, I use regularly. What option do I select in CLA?

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
> @ItzLevvie2 CLA says gives two options to agree; either I'm the owner or submitting on behalf of an employer. But neither is the case. I just wanted to submit this app, I use regularly. What option do I select in CLA?

If you're just a user submitting an application you use regularly, choose `I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.`

'Ownership' here means the package you're creating and/or submitting into this repository.

It doesn't have anything to do with the rights of the installer or application.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
You also have some changes to make in this PR: https://github.com/microsoft/winget-pkgs/pull/34108#pullrequestreview-799326910

You need to change the version from `5.0.22.0` to `5.0` in order to follow the version number from the Control Panel, otherwise, the `winget upgrade ConceptworldCorporation.RecentX` command will end up in an upgrade loop.

You'll also have to update the version folder from `5.0.22.0` to `5.0` which you can do so by pressing the `.` (dot) on your keyboard in this PR.

**Comment by ItzLevvie** [comment]
Instructions:

1) Click on Explorer:
image

2) Find the files you created or modified in this PR and click on the "Rename" button:
image

3) Write `5.0` instead of `5.0.22.0`:
image

4) You may need to commit the changes to your PR (if it's not done automatically) by clicking on Source Control:
image

5) Write a commit message in the box:
image

6) Then click on the checkmark to commit your changes:
image

**Review by heymaad — COMMENTED** [review]
Rename changes approved. @ItzLevvie Is it correct now?

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**